hack/tools/release_notes.go

```diff
@@ -42,6 +42,7 @@ const (
 	warning       = ":warning: Breaking Changes"
 	other         = ":seedling: Others"
 	unknown       = ":question: Sort these by hand"
+	superseded    = ":arrow_up: Superseded or Reverted"
```

Comment:
nit/question: Why :arrow_up:? It seems more like "upgrade" to me. Maybe :rewind: would fit better?

Reply:
Most of them are superseded bumps, bumps go up in my mind. :smile: 

I'm open to other icons, but prefer not defaulting to << which is revert focused (again, to my context).

Reply:
We could go for :ghost: or :skull: (dead PRs), :back: (back'd away), :arrow_backward: or  ⏪ (revert first) etc

Reply:
Hehe I like :ghost: ! :dash: (gone already) could maybe work? No strong opinions from me though, I was just curious about the :arrow_up:

Reply:
I agree that there are better options than :arrow_up:, it just felt relevant at the time.
@elfosardo @zaneb @dtantsur @honza feel free to throw in your 2 cents as well.

Reply:
Changed to :recycle: after discussion.